charts/argo-cd/Chart.yaml

```diff
@@ -1,8 +1,8 @@
-apiVersion: v1
-appVersion: 1.8.4
+apiVersion: v2
+appVersion: 2.0.0
 description: A Helm chart for ArgoCD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
```

Comment:
Thanks for taking this opportunity to change to apiVersion v2. Please note, that we also need to change the dependency model for the redis-ha chart.

I need you to delete thw following two files:
  *  requirements.lock
  * requirements.yaml

In the `Chart.yaml` we need to add the `dependencies` section (replaces the requirement files):

```
dependencies:
  - name: redis-ha
    version: 4.10.1
    repository: https://dandydeveloper.github.io/charts/
    condition: redis-ha.enabled
```
You can also see this changes in my pr - https://github.com/argoproj/argo-helm/pull/614

After that update the dependencies (`helm dependency update`) and remove the charts folder. (We need to update the Chart.lock).

Finally, we need to add a new section in the `Upgrading` heading. Something like this: 

``` 
3.0.0 and above
Helm apiVersion switched to `v2`. Requires Helm `3.0.0` or above to install. [Read More](https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/) on how to migrate your release from Helm 2 to Helm 3.
```
Since this is the effective change. I don't think your other changes require manual interaction, therefor we don't need to mention them.

If you have done these changes we can approve this pull request. If you need help, let me know. Thanks again.